### ISSUE TITLE
Inserting pages in first level folders leads to error

### ISSUE THREAD

**Issue by fritzmg** [terminal42/contao-folderpage#21]
If you try to insert a new page into this _Test_ folder:

<img src="https://user-images.githubusercontent.com/4970961/132470573-426ed98f-dbfa-4d6f-bb39-371691bb39a8.png" width="150">

the default type of the new page will be `root` and if you then try to change it to `regular`, the following error will appear:

```
Symfony\Component\Routing\Exception\InvalidParameterException:
Parameter "_locale" for route "contao_frontend" must match "[a-z]{2}(?:\-[A-Z]{2})?" ("" given) to generate a corresponding URL.

  at vendor\symfony\routing\Generator\UrlGenerator.php:193
  at Symfony\Component\Routing\Generator\UrlGenerator->doGenerate(array('_locale' => 0, 'alias' => 1), array('_token_check' => true, '_controller' => 'Contao/CoreBundle/Controller/FrontendController::indexAction', '_scope' => 'frontend'), array('alias' => '.+', '_locale' => '[a-z]{2}(/-[A-Z]{2})?'), array(array('text', '.html'), array('variable', '/', '.+', 'alias'), array('variable', '/', '[a-z]{2}(?:/-[A-Z]{2})?', '_locale')), array('_locale' => '', 'alias' => '02edbe5cce83efaf6880'), 'contao_frontend', 0, array())
     (vendor\symfony-cmf\routing\src\ProviderBasedGenerator.php:78)
  at Symfony\Cmf\Component\Routing\ProviderBasedGenerator->generate('contao_frontend', array('_locale' => '', 'alias' => '02edbe5cce83efaf6880'), 0)
     (vendor\symfony-cmf\routing\src\DynamicRouter.php:186)
  at Symfony\Cmf\Component\Routing\DynamicRouter->generate('contao_frontend', array('_locale' => '', 'alias' => '02edbe5cce83efaf6880'), 0)
     (vendor\symfony-cmf\routing\src\ChainRouter.php:248)
  at Symfony\Cmf\Component\Routing\ChainRouter->generate('contao_frontend', array('_locale' => '', 'alias' => '02edbe5cce83efaf6880'), 0)
     (vendor\contao\contao\core-bundle\src\Routing\UrlGenerator.php:81)
  at Contao\CoreBundle\Routing\UrlGenerator->generate('02edbe5cce83efaf6880', array('_locale' => '', 'alias' => '02edbe5cce83efaf6880'), 0)
     (vendor\contao\contao\core-bundle\src\Resources\contao\models\PageModel.php:1134)
  at Contao\PageModel->getAbsoluteUrl()
     (vendor\contao\contao\core-bundle\src\Resources\contao\dca\tl_page.php:996)
  at tl_page->getSerpUrl(object(PageModel))
     (vendor\contao\contao\core-bundle\src\Resources\contao\widgets\SerpPreview.php:150)
  at Contao\SerpPreview->getUrl(object(PageModel))
     (vendor\contao\contao\core-bundle\src\Resources\contao\widgets\SerpPreview.php:47)
  at Contao\SerpPreview->generate()
     (vendor\contao\contao\core-bundle\src\Resources\contao\library\Contao\Widget.php:651)
  at Contao\Widget->generateWithError(true)
     (vendor\contao\contao\core-bundle\src\Resources\contao\templates\backend\be_widget.html5:3)
  at include('vendor/contao/contao/core-bundle/src/Resources/contao/templates/backend/be_widget.html5')
     (vendor\contao\contao\core-bundle\src\Resources\contao\library\Contao\TemplateInheritance.php:100)
  at Contao\Widget->inherit()
     (vendor\contao\contao\core-bundle\src\Resources\contao\library\Contao\Widget.php:598)
  at Contao\Widget->parse()
     (vendor\contao\contao\core-bundle\src\Resources\contao\classes\DataContainer.php:668)
  at Contao\DataContainer->row('{title_legend},title,alias,type;{meta_legend},pageTitle,robots,description,serpPreview;{protected_legend:hide},protected;{layout_legend:hide},includeLayout;{cache_legend:hide},includeCache;{chmod_legend:hide},includeChmod;{expert_legend:hide},cssClass,sitemap,hide,noSearch,guests,requireItem;{tabnav_legend:hide},tabindex,accesskey;{publish_legend},published,start,stop')
     (vendor\contao\contao\core-bundle\src\Resources\contao\drivers\DC_Table.php:1985)
  at Contao\DC_Table->edit()
     (vendor\contao\contao\core-bundle\src\Resources\contao\classes\Backend.php:644)
  at Contao\Backend->getBackendModule('page', null)
     (vendor\contao\contao\core-bundle\src\Resources\contao\controllers\BackendMain.php:167)
  at Contao\BackendMain->run()
     (vendor\contao\contao\core-bundle\src\Controller\BackendController.php:48)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (vendor\symfony\http-kernel\HttpKernel.php:158)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor\symfony\http-kernel\HttpKernel.php:80)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor\symfony\http-kernel\Kernel.php:201)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (web\index.php:31)
  at require('web/index.php')
     (web\app.php:4)         
```

**Comment by aschempp** [comment]
I don't think this is an issue of this extension, I can reproduce in on demo.contao.org

**Comment by fritzmg** [comment]
How did you reproduce this? If I try to insert a new page into another page in the demo, the type will be `default`, not `root`. And the subsequent error will also not occur.

The `root` default value is forced here: https://github.com/terminal42/contao-folderpage/blob/ad54d439dbfa268b52268d8782443599206ad029/src/DataContainer/Page.php#L263-L282

**Comment by aschempp** [comment]
I have fixed the incorrect type in 111cd982f402c3b8045e42b1f97945efcaaad6f1 but I was unable to reproduce the error. The message itself results from Contao (the SERP widget), not the extension.
